pkg/plugins/models.go

```diff
@@ -314,6 +314,7 @@ func (e Error) PublicMessage() string {
 }
 
 // Access-Control related definitions
+const ActionAppAccess = "plugins.app:access"
```

Comment:
Can I import the `pluginaccesscontrol` package instead of redefining here, or is it annoying as you want to keep the package as isolated as possible?

Reply:
`pluginaccesscontrol` is in `pkg/services/pluginsintegration/pluginaccesscontrol` and the idea is that pluginintegration packages implements parts of pkg/plugins and in addition it imports below dependencies so would advice against it
```
ac "github.com/grafana/grafana/pkg/services/accesscontrol"
	contextmodel "github.com/grafana/grafana/pkg/services/contexthandler/model"
	"github.com/grafana/grafana/pkg/services/featuremgmt"
	"github.com/grafana/grafana/pkg/services/org"
```

If you want the definitions to be sharable would advice putting them in pkg/plugins or something similar